src/xform_to_datalog/testdata/ok_claim_propagates.dl

```diff
@@ -0,0 +1,49 @@
+// GENERATED FILE, DO NOT EDIT!
+
+#include "taint.dl"
+
+// Rules for detecting policy failures.
```

Comment:
Did you mean to check in this file containing the Datalog generated from the `ok_claim_propagates` pieces?

Reply:
Yes, it is used in `generate_datalog_program_test.sh` as a golden file.